cockroachdb/tests/conftest.py

```diff
@@ -21,6 +21,7 @@ def dd_environment(instance):
         os.path.join(DOCKER_DIR, 'docker-compose.yaml'),
         env_vars=env_vars,
         endpoints=instance['openmetrics_endpoint'],
+        # conditions=[run_sql], # Uncomment to run the sql script in the cockroachdb container. See /tests/README.md.
```

Comment:
Would there be any way to avoid having to uncomment / comment to enable or disable this? Maybe we can have this depend on an environment variable and perhaps set up a hatch environment (that doesn't become part of the testing matrix) that lets us spin up this environment without having to modify any code? I don't know if it's possible though.

Reply:
let's try the env variable route for now